<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Curves</class>
 <widget class="QWidget" name="Curves">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="curvesLayout"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_1">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGraphicsView" name="curvePreview">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>400</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="curveNameLabel">
         <property name="text">
          <string>Curve name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="curvePoints"/>
         </item>
         <item>
          <widget class="QComboBox" name="curveCustom">
           <item>
            <property name="text">
             <string>Fixed X</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Custom X</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="curveSmooth">
           <item>
            <property name="text">
             <string>Lines</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Smooth</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="curveName">
         <property name="maxLength">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="curveTypeLabel">
         <property name="text">
          <string>Curve type</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="curvesLayout2"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QGridLayout" name="pointsLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>0</number>
       </property>
       <property name="verticalSpacing">
        <number>2</number>
       </property>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="curveCreator">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Curve Creator</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_76" rowstretch="0,0,0,0,0,0,0,0,0,0">
      <item row="1" column="0">
       <widget class="QLabel" name="curveCoeffLabel">
        <property name="text">
         <string>Coefficient</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="curveTypeLabel_2">
        <property name="text">
         <string>Curve type</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="curveSideLabel">
        <property name="text">
         <string>Side</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QSpinBox" name="yMax">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QSpinBox" name="yMid">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="yMinLabel">
        <property name="text">
         <string>Y at X=-100</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="3">
       <widget class="QComboBox" name="curveSide">
        <item>
         <property name="text">
          <string>Both</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>x&gt;0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>x&lt;0</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="yMaxLabel">
        <property name="text">
         <string>Y at X=100</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QSpinBox" name="curveCoeff">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="4">
       <widget class="QPushButton" name="curveApply">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QComboBox" name="curveType"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="yMidLabel">
        <property name="text">
         <string>Y at X=0</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="pointSizeLabel">
        <property name="text">
         <string>Point size</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QSpinBox" name="yMin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="3">
       <widget class="QSpinBox" name="pointSize"/>
      </item>
      <item row="7" column="0" colspan="4">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>curvePreview</tabstop>
  <tabstop>curvePoints</tabstop>
  <tabstop>curveCustom</tabstop>
  <tabstop>curveSmooth</tabstop>
  <tabstop>curveName</tabstop>
  <tabstop>curveType</tabstop>
  <tabstop>curveCoeff</tabstop>
  <tabstop>yMin</tabstop>
  <tabstop>yMid</tabstop>
  <tabstop>yMax</tabstop>
  <tabstop>curveSide</tabstop>
  <tabstop>curveApply</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
